sal_interfaces/ATWhiteLight/ATWhiteLight_Telemetry.xml

```diff
@@ -3,202 +3,117 @@
+      <EFDB_Name>fan4</EFDB_Name>
       <Description>fan 4 speed</Description>
-      <IDL_Type>long</IDL_Type>
+      <IDL_Type>float</IDL_Type>
       <Units>Hz</Units>
       <Count>1</Count>
     </item>
-    <item>
-      <EFDB_Name>timestamp</EFDB_Name>
-      <Description>Time of measurement (TAI unix seconds)</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>second</Units>
-      <Count>1</Count>
-    </item>
-  </SALTelemetry>
-  <SALTelemetry>
-    <Subsystem>ATWhiteLight</Subsystem>
-    <EFDB_Topic>ATWhiteLight_chillerUpTime</EFDB_Topic>
-    <item>
-      <EFDB_Name>upTime</EFDB_Name>
-      <Description>chiller uptime</Description>
-      <IDL_Type>long</IDL_Type>
-      <Units>minute</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>timestamp</EFDB_Name>
-      <Description>Time of measurement (TAI unix seconds)</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>second</Units>
-      <Count>1</Count>
-    </item>
   </SALTelemetry>
   <SALTelemetry>
     <Subsystem>ATWhiteLight</Subsystem>
-    <EFDB_Topic>ATWhiteLight_chillerTempSensors</EFDB_Topic>
+    <EFDB_Topic>ATWhiteLight_chillerTemperatures</EFDB_Topic>
+    <Description>Chiller set temperature and measured temperatures</Description>
     <item>
       <EFDB_Name>setTemperature</EFDB_Name>
-      <Description>the celsius temperature we currently targetting</Description>
+      <Description>Set temperature</Description>
       <IDL_Type>float</IDL_Type>
-      <Units>degree</Units>
+      <Units>deg_C</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>supplyTemperature</EFDB_Name>
-      <Description>the celsius temperature of the coolant supply</Description>
+      <Description>Supply coolant temperature</Description>
       <IDL_Type>float</IDL_Type>
-      <Units>degree</Units>
+      <Units>deg_C</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>returnTemperature</EFDB_Name>
-      <Description>the celsius return temperature of coolant</Description>
+      <Description>Return coolant temperature</Description>
       <IDL_Type>float</IDL_Type>
-      <Units>degree</Units>
+      <Units>deg_C</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>ambientTemperature</EFDB_Name>
-      <Description>ambient temperature celsius</Description>
+      <Description>Ambient temperature</Description>
       <IDL_Type>float</IDL_Type>
-      <Units>degree</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>timestamp</EFDB_Name>
-      <Description>Time of measurement (TAI unix seconds)</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>second</Units>
+      <Units>deg_C</Units>
       <Count>1</Count>
     </item>
   </SALTelemetry>
   <SALTelemetry>
     <Subsystem>ATWhiteLight</Subsystem>
     <EFDB_Topic>ATWhiteLight_chillerProcessFlow</EFDB_Topic>
+    <Description>Chiller process flow rate</Description>
```

Comment:
Good point and I fully agree with you. It seems to be a quirk of the ThermoTek terminology. Perhaps coolant flow would be better. I think it might be good to use some modifier of "flow", since otherwise a user might think it is airflow.